.travis/cherrypick.py

```diff
@@ -97,14 +97,14 @@ def get_merged_commits(pr):
 
 # check if we cherry picked anything
 if len(cherrypicks) == 0:
-    print(f"No cherry picks detected.")
+    print("No cherry picks detected.")
```

Comment:
We need to change it on `plugin_template` or  add `./.travis` on flake8 ignore list

Reply:
:+1: I'll open a pr.

Reply:
https://github.com/pulp/plugin_template/pull/216